examples/alexNet/Labels.hs

```diff
@@ -0,0 +1,1007 @@
+module Labels (labels) where
+
+import Data.Array
+
+labels = array (0, 999) $ zip [0 .. 999] list
+
+list = 
```

Comment:
I'm confused. Why do we have this huge file now?

Reply:
Actually, the file contains the labels of 1000 classes from ImageNet dataset. For now, I have placed it in the alexNet directory maybe later we can move it to Torch.Vision, or if unnecessary remove it.

Reply:
Would prefer to keep data files out of the repo and out of the history as well so may need to rebase. We can have scripts to download data and metadata as needed, but let's try not to have dataset contents themselves in the repo unless there's a good reason.

Reply:
Also, it's usually good practice to avoid adding things "just in case", because they might not ever end up being useful and you can accumulate a bunch of dead code this way. Just something to keep in mind for the future!

Reply:
Thank you for your suggestion, I will abide by it.